app/templates/index.hbs

```diff
@@ -0,0 +1 @@
+{{outlet}}
```

Comment:
Don't need this file - I'll merge this in but make sure it's gone by the next PR.